src/main/java/com/dev/cinema/dao/impl/GenericDaoImpl.java

```diff
@@ -3,12 +3,15 @@
 import com.dev.cinema.dao.GenericDao;
 import com.dev.cinema.exeption.DataProcessingExeption;
 import com.dev.cinema.util.HibernateUtil;
+import lombok.extern.log4j.Log4j;
 import org.hibernate.Session;
 import org.hibernate.Transaction;
 
+@Log4j
 public abstract class GenericDaoImpl<T> implements GenericDao<T> {
     @Override
     public T add(T entity) {
+        log.info("Сalling method add for " + entity);
```

Comment:
let's add logging about successfully added entity(when it's added)

Reply:
Added.